src/extensions.css

```diff
@@ -157,7 +158,8 @@
   }
   /* Sourcegraph */
   /** CodeLine Hover Color In Blob **/
+  /** https://github.com/StylishThemes/GitHub-Dark/pull/1216 **/
```

Comment:
You also dont need to add a reference to the PR, the comments above this one are enough.